[2024-05-19T20:26:51.505+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-19T20:26:51.516+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: silver_s3.process_parquet_files scheduled__2023-05-23T00:00:00+00:00 [queued]>
[2024-05-19T20:26:51.521+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: silver_s3.process_parquet_files scheduled__2023-05-23T00:00:00+00:00 [queued]>
[2024-05-19T20:26:51.521+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-05-19T20:26:51.530+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): process_parquet_files> on 2023-05-23 00:00:00+00:00
[2024-05-19T20:26:51.538+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=771) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-19T20:26:51.541+0000] {standard_task_runner.py:63} INFO - Started process 785 to run task
[2024-05-19T20:26:51.541+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'silver_s3', 'process_parquet_files', 'scheduled__2023-05-23T00:00:00+00:00', '--job-id', '272', '--raw', '--subdir', 'DAGS_FOLDER/silver_s3.py', '--cfg-path', '/tmp/tmpj56lr93z']
[2024-05-19T20:26:51.546+0000] {standard_task_runner.py:91} INFO - Job 272: Subtask process_parquet_files
[2024-05-19T20:26:51.589+0000] {task_command.py:426} INFO - Running <TaskInstance: silver_s3.process_parquet_files scheduled__2023-05-23T00:00:00+00:00 [running]> on host 620090466bc8
[2024-05-19T20:26:51.648+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='silver_s3' AIRFLOW_CTX_TASK_ID='process_parquet_files' AIRFLOW_CTX_EXECUTION_DATE='2023-05-23T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-05-23T00:00:00+00:00'
[2024-05-19T20:26:51.649+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-19T20:26:51.665+0000] {base.py:84} INFO - Using connection ID 'aws_s3' for task execution.
[2024-05-19T20:26:51.667+0000] {connection_wrapper.py:384} INFO - AWS Connection (conn_id='aws_s3', conn_type='aws') credentials retrieved from login and password.
[2024-05-19T20:26:56.332+0000] {logging_mixin.py:188} INFO - No .parquet files found.
[2024-05-19T20:26:56.333+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-19T20:26:56.334+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-19T20:26:56.342+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=silver_s3, task_id=process_parquet_files, run_id=scheduled__2023-05-23T00:00:00+00:00, execution_date=20230523T000000, start_date=20240519T202651, end_date=20240519T202656
[2024-05-19T20:26:56.384+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-19T20:26:56.401+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-19T20:26:56.402+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2024-05-19T20:37:09.040+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-19T20:37:09.048+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: silver_s3.process_parquet_files scheduled__2023-05-23T00:00:00+00:00 [queued]>
[2024-05-19T20:37:09.053+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: silver_s3.process_parquet_files scheduled__2023-05-23T00:00:00+00:00 [queued]>
[2024-05-19T20:37:09.054+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-05-19T20:37:09.062+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): process_parquet_files> on 2023-05-23 00:00:00+00:00
[2024-05-19T20:37:09.066+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=2049) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-19T20:37:09.068+0000] {standard_task_runner.py:63} INFO - Started process 2061 to run task
[2024-05-19T20:37:09.068+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'silver_s3', 'process_parquet_files', 'scheduled__2023-05-23T00:00:00+00:00', '--job-id', '388', '--raw', '--subdir', 'DAGS_FOLDER/silver_s3.py', '--cfg-path', '/tmp/tmpkgy82ado']
[2024-05-19T20:37:09.069+0000] {standard_task_runner.py:91} INFO - Job 388: Subtask process_parquet_files
[2024-05-19T20:37:09.095+0000] {task_command.py:426} INFO - Running <TaskInstance: silver_s3.process_parquet_files scheduled__2023-05-23T00:00:00+00:00 [running]> on host 620090466bc8
[2024-05-19T20:37:09.138+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='silver_s3' AIRFLOW_CTX_TASK_ID='process_parquet_files' AIRFLOW_CTX_EXECUTION_DATE='2023-05-23T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-05-23T00:00:00+00:00'
[2024-05-19T20:37:09.139+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-19T20:37:09.162+0000] {base.py:84} INFO - Using connection ID 'aws_s3' for task execution.
[2024-05-19T20:37:09.163+0000] {connection_wrapper.py:384} INFO - AWS Connection (conn_id='aws_s3', conn_type='aws') credentials retrieved from login and password.
[2024-05-19T20:37:18.045+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-19T20:37:18.054+0000] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 400, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 235, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 252, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/dags/silver_s3.py", line 69, in process_parquet_files
    os.remove(file_path)
FileNotFoundError: [Errno 2] No such file or directory: '/tmp/disney_data_20240518T000000.parquet'
[2024-05-19T20:37:18.085+0000] {taskinstance.py:1206} INFO - Marking task as UP_FOR_RETRY. dag_id=silver_s3, task_id=process_parquet_files, run_id=scheduled__2023-05-23T00:00:00+00:00, execution_date=20230523T000000, start_date=20240519T203709, end_date=20240519T203718
[2024-05-19T20:37:18.103+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 388 for task process_parquet_files ([Errno 2] No such file or directory: '/tmp/disney_data_20240518T000000.parquet'; 2061)
[2024-05-19T20:37:18.158+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2024-05-19T20:37:18.169+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-19T20:37:18.170+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
